### DISCUSSION TITLE
Is there a way to generate and send invoices automatically?

### DISCUSSION THREAD

**Discussion by ex5** [saleor/saleor#9889] [Q&A]
In the Dashboard, there are `Generate` and `Send` buttons for invoices:
![2022-05-30-172657_450x176_scrot](https://user-images.githubusercontent.com/176934/171023408-0fd3358b-6811-474f-89ec-7f3207b6d968.png)

However, normally a shop would generate and send out an invoice as soon as the payment for an order is captured.
A something similar to a cronjob would do, but I don't see any Django commands related to invoice generation that could be used for this.

Is there a recommended way to automate this?

**Comment by karolkielecki** [comment]
Hi @ex5,

I think you will need to develop an invoicing app. You can find more info here: https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts
